broadcasting/09_ConnectionManager.md

```diff
@@ -0,0 +1,71 @@
+Now let's focus on how the Connection Manager works. As mentioned previously its role is to establish the RTSP connection with the RTSP server.
+
+The `ConnectionManager` module will use the [`Connection`](https://hexdocs.pm/connection/Connection.html) behaviour, which provides additional callbacks to [`GenServer`](https://hexdocs.pm/elixir/GenServer.html) behaviour, aiding with building a connection process.
+
+First of all we are defining the `ConnectionStatus` struct, which we will use to keep the state of the ConnectionManager:
+
+##### lib/connection_manager.ex
+```elixir
+defmodule ConnectionStatus do
+  @moduledoc false
+  @type t :: %__MODULE__{
+          stream_url: binary(),
+          rtsp_session: pid(),
+          pipeline: pid(),
+          keep_alive: pid(),
+          pipeline_options: keyword()
+          }
+
+  @enforce_keys [
+      :stream_url,
+      :pipeline,
+      :pipeline_options
+  ]
+
+  defstruct @enforce_keys ++
+              [
+                  :rtsp_session,
+                  :keep_alive
+              ]
+end
+```
+
+It holds the `rtsp_session`, which is the pid of a process started with `Membrane.RTSP.start/1`. The `Membrane.RTSP` allows us to execute RTSP client commands. You can read more about  [here](https://hexdocs.pm/membrane_rtsp/readme.html).
+The `pipeline` field is the pid of the pipeline, we will need it to notify the pipeline, that the RTSP connection is ready together with `pipeline_options`, which contain necessary information about the stream.
```

Comment:
'...together with the `pipeliene_options` ' - as far as I am concerned, I would say that that sentence tells us, that it is the RTSP connection that is "ready with the `pipeline_options`" - though, I believe, that it should mean, that we will be notyfing the pipeline with the information that RTSP is ready, together with the `pipeline_options` .
I would consider rewritting that sentence to make it sound more clear.

Reply:
done